flixel/addons/tile/FlxTilemapExt.hx

```diff
@@ -539,6 +524,79 @@ class FlxTilemapExt extends FlxTilemap
 	}
 	
 	/**
+	 * Sets the slope arrays, which define which tiles are treated as slopes of 22.5 degrees.
+	 * 
+	 * @param 	HighSlopes 	An array containing the numbers of the tiles to be treated as high slope tiles with a slope on the left.
+	 * @param 	LowSlopes	An array containing the numbers of the tiles to be treated as low slope tiles with a slope on the right.
+	 */
+	public function setSlopes22(?HighSlopes:Array<Int>, ?LowSlopes:Array<Int>) 
```

Comment:
The user must set the slops of 22.5 and 67.5 degrees in the `setSlopes` method as the default ones.

But he need to identify if it is a 22 of 67 and if is a High or a Low slope.
I think the names of functions and of params are not yet intuitive enough.
![slopenames](https://cloud.githubusercontent.com/assets/13324869/10081076/a03aa350-62c9-11e5-89b0-390f22fa5e8e.png)

Need suggestions here...